.yamato/hdrp_hybrid-osx-metal.yml

```diff
@@ -168,7 +168,7 @@ HDRP_Hybrid_OSX_Metal_Standalone_mono_apiNet2_Linear_CUSTOM-REVISION:
       -  ssh -i ~/.ssh/id_rsa_macmini -o "StrictHostKeyChecking=no" bokken@$BOKKEN_DEVICE_IP '$(/usr/local/bin/python3 -m site --user-base)/bin/unity-downloader-cli --source-file ~/Graphics/unity_revision.txt -c editor -c il2cpp  --wait --published-only'
       -  |5-
 
-                 ssh -i ~/.ssh/id_rsa_macmini -o "StrictHostKeyChecking=no" bokken@$BOKKEN_DEVICE_IP "export UPM_REGISTRY=https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates; echo \$UPM_REGISTRY; cd ~/Graphics/TestProjects/HDRP_HybridTests && ~/Graphics/TestProjects/HDRP_HybridTests/utr --artifacts_path=/Users/bokken/Graphics/TestProjects/HDRP_HybridTests/test-results --compilation-errors-as-warnings --editor-location=/Users/bokken/.Editor --extra-editor-arg="-colorspace=Linear" --platform=StandaloneOSX --reruncount=2 --scripting-backend=Mono2x --suite=playmode --testproject=/Users/bokken/Graphics/TestProjects/HDRP_HybridTests --timeout=2400"
+                 ssh -i ~/.ssh/id_rsa_macmini -o "StrictHostKeyChecking=no" bokken@$BOKKEN_DEVICE_IP "export UPM_REGISTRY=https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates; echo \$UPM_REGISTRY; cd ~/Graphics/TestProjects/HDRP_HybridTests && ~/Graphics/TestProjects/HDRP_HybridTests/utr --artifacts_path=/Users/bokken/Graphics/TestProjects/HDRP_HybridTests/test-results --compilation-errors-as-warnings --editor-location=/Users/bokken/.Editor --platform=StandaloneOSX --reruncount=2 --suite=playmode --testproject=/Users/bokken/Graphics/TestProjects/HDRP_HybridTests --timeout=2400 --timeout=1200"
```

Comment:
same here with colorspace and scripting backend (and other platforms too), and seems like we get 2x timeout instead of last one overwriting?

Reply:
im not actually sure why this is happening 🤔

Reply:
fixed the removed scripting-backend, still confused about timeout